examples/serviceconfig.json

```diff
@@ -1,15 +1,16 @@
 {
-  "disable_persistent_config": true,
-  "adminInterface": "127.0.0.1:4985",
-  "interface": "0.0.0.0:4984",
-  "databases": {
-    "db": {
-      "server": "walrus:data",
-      "users": {
-        "GUEST": {"disabled": false, "admin_channels": ["*"] }
-      },
-      "allow_conflicts": false,
-      "revs_limit": 20
+  "bootstrap": {
+    "server": "rosmar:/?mode=memory"
+  },
+  "api": {
+    "public_interface": ":4984",
+    "admin_interface": ":4985"
```

Comment:
We should definitely keep the existing interface setting which locked down the admin API by default.

I'm OK with dropping `0.0.0.0` for the public interface though.

Reply:
Will add that now! 

Just thought should I be adding admin_interface_authentication: false as this is rosmar config?